doc/release-notes.md

```diff
@@ -36,49 +36,429 @@ reindex or re-sync the whole chain.
 
 # Notable changes
 
-## CoinJoin improvements
+## ChainLock-based random beacon
+
+To improve long-living masternode quorum (LLMQ) selection, the `v20` hard fork
+will enable a new random beacon using ChainLock signatures as described in
+[DIP-0029](https://github.com/dashpay/dips/blob/master/dip-0029.md) (__TODO__: NOT MERGED).
+ChainLock signatures will be stored in the coinbase transaction to be used as a
+source of randomness during quorum member calculations. This change increases
+the decentralization in the source of randomness and prevents masternodes or
+miners from being able to manipulate the LLMQ process.
+
+## Masternode reward reallocation
```

Comment:
No longer v20

Reply:
technically speaking, it was implemented :) but yeah, won't be enabled yet.